travis/licenses_golden/licenses_third_party

```diff
@@ -23889,6 +23889,79 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 
 ====================================================================================================
+LIBRARY: pkg
+ORIGIN: ../../../dart/third_party/observatory_pub_packages/packages/async/LICENSE
+TYPE: LicenseType.bsd
+FILE: ../../../third_party/pkg/when/.travis.yml
+FILE: ../../../third_party/pkg/when/example/foo.json
+FILE: ../../../third_party/pkg/when/example/read_json_file.dart
+----------------------------------------------------------------------------------------------------
+Copyright 2015, the Dart project authors. All rights reserved.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are
+met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above
+      copyright notice, this list of conditions and the following
+      disclaimer in the documentation and/or other materials provided
+      with the distribution.
+    * Neither the name of Google Inc. nor the names of its
+      contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+====================================================================================================
+
+====================================================================================================
+LIBRARY: pkg
+ORIGIN: ../../../third_party/pkg/when/lib/when.dart + ../../../dart/third_party/observatory_pub_packages/packages/async/LICENSE
+TYPE: LicenseType.bsd
+FILE: ../../../third_party/pkg/when/lib/when.dart
+----------------------------------------------------------------------------------------------------
+Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+for details. All rights reserved. Use of this source code is governed by a
+BSD-style license that can be found in the LICENSE file
```

Comment:
Looking at the patterns (https://github.com/flutter/engine/blob/master/tools/licenses/lib/patterns.dart#L293), it looks like the problem is that for some reason these copyrights say "Please see the AUTHORS
file for details" before they say "All rights reserved". Not sure why, it might be worth sending a query upstream to see if that was approved by the relevant lawyers (it's not what I would expect from Google code). But in any case the fix is just to add another pattern there in that file that flips the two sentences around accordingly.